### DISCUSSION TITLE
API permissions -- Everything

### DISCUSSION THREAD

**Discussion by justdaj** [mastodon/mastodon#23530] [Q&A]
Hi all,
Quick questions.  When an Application is asking for "Everything - Read only" to a Mastodon user, what exactly is everything?

- Is it as **ALL** posts (Public, Unlisted, Followers Only, Mentioned People only)
- Your profile
- A list of who you follow and who follows you?

I can't find anything in the documentation defining the scope of "Everything"

thanks
David

**Comment by trwnh** [comment]
the universal `read` scope grants access to read all posts, either as an application or optionally with some authorized user. app tokens can read public/unlisted posts, and user tokens can read any post they have permission to see. profiles can always be read, including any available information on follows/followers (unless they hide their network).

basically, if you are seeing this authorization request as a user, then yes, it includes all of those things you listed -- "everything".
